WebRoot/js/zimbraMail/calendar/view/ZmApptEditView.js

```diff
@@ -1520,7 +1522,7 @@ function(locationExceptions, alteredLocations) {
 ZmApptEditView.prototype._toggleOptionalAttendees =
 function(forceShow) {
     this._optionalAttendeesShown = ! this._optionalAttendeesShown || forceShow;
-    this._showOptional.innerHTML = this._optionalAttendeesShown ? ZmMsg.hideOptional : ZmMsg.showOptional;
+    Dwt.setVisible(this._showOptional, !Boolean(this._optionalAttendeesShown));
```

Comment:
Boolean('false') will be converted to true, not sure if there is possibility of having strings in _optionalAttendeesShown

Reply:
If you look at line#1524 above, we are casting it to `Boolean` before using it as line#1525.  (Please see negate `!` operator before it), so I don't see possibility of having string here.

Let me know if you have any question.

Thanks

Reply:
ok thanks